packages/tendermint-rpc/src/adaptors/v0-33/hasher.ts

```diff
@@ -1,4 +1,4 @@
-import { Sha256 } from "@cosmjs/crypto";
+import { Sha256, sha256 } from "@cosmjs/crypto";
```

Comment:
Not strictly part of this issue, but since we're tidying things up, maybe you want to refactor the other place where the `Sha256` class is used.

Reply:
The other use cases in this file require incremental hashing, which is why `Sha256` is still imported here